docs/maps/import-geospatial-data.asciidoc

```diff
@@ -0,0 +1,30 @@
+[role="xpack"]
+[[import-geospatial-data]]
+== Import geospatial data
+
+In order to use your geospatial data in the Elastic Stack, it must be indexed as {ref}/geo-point.html[geo_point] or {ref}/geo-shape.html[geo_shape].
+Kibana makes it easy to import geospatial data into the Elastic Stack.
+
+=== Upload a CSV file with latitude and longitude columns
```

Comment:
Add [discrete] above this title so that content appears on the same page.

Can you shorten this title?

Reply:
> Add [discrete] above this title so that content appears on the same page.

I have also seen this done with `[float]`. What is the difference between `[discrete]` and `[float]`?

> Can you shorten this title?

How about "Upload CSV with latitude and longitude columns"?